fhir/com.b2international.snowowl.fhir.core/src/com/b2international/snowowl/fhir/core/model/codesystem/CodeSystem.java

```diff
@@ -149,63 +162,45 @@
 	private Integer count;
 
 	@Summary
-	@Valid
+	@JsonProperty(CodeSystem.Fields.FILTER)
+	@JsonInclude(value = Include.NON_EMPTY)
 	private Collection<Filter> filters;
 
 	/*
 	 * The properties supported by this code system
 	 */
 	@Summary
 	@Valid
+	@JsonProperty(CodeSystem.Fields.PROPERTY)
+	@JsonInclude(value = Include.NON_EMPTY)
 	private Collection<SupportedConceptProperty> properties;
 
 	/*
 	 * Concepts in the code system, up to the server if they are returned
 	 */
 	@Valid
+	@JsonProperty("concept")
+	@JsonInclude(value = Include.NON_EMPTY)
```

Comment:
I think empty arrays should be returned when requested to indicate that there are no filters/concepts/properties/etc.